packages/example/src/example.ts

```diff
@@ -12,14 +12,71 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { createEnv } from "@bufbuild/cel";
-import { createRegistry } from "@bufbuild/protobuf";
+import {
+  celEnv,
+  CelScalar,
+  Func,
+  FuncOverload,
+  parse,
+  plan,
+  run,
+} from "@bufbuild/cel";
+import { STRINGS_EXT_FUNCS } from "@bufbuild/cel/ext/strings";
 
-const env = createEnv("", createRegistry());
-env.set("name", "tacocat");
+// Run a CEL expression:
```

Comment:
Nice!

If we add type checking, we should infer variable types from the declaration. Will be interesting to see how this works out without breaking this API, but we can handle it if we get there.